src/UserManagerEvents.ts

```diff
@@ -163,4 +163,5 @@ export class UserManagerEvents extends AccessTokenEvents {
     public _raiseUserSessionChanged(): void {
         this._userSessionChanged.raise();
     }
+
```

Comment:
please avoid changing code style,

Reply:
I am sorry, this is something that my IDE does if not otherwise configured. Perhaps you should consider enforcing this specific rule as well via the .editorconfig file if possible.